annotated_test.go

```diff
@@ -477,6 +480,32 @@ func TestAnnotatedAs(t *testing.T) {
 				assert.Equal(t, s.String(), "another stringer")
 			},
 		},
+		{
+			desc: "value type convertible to target type",
```

Comment:
nit: consider downscoping the names of these cases since convertible types are locked down to just functions right now
```suggestion
			desc: "function value convertible to target type",
```